src/main/java/org/brapi/test/BrAPITestServer/model/entity/core/ListEntity.java

```diff
@@ -97,6 +99,10 @@ public void setListType(ListTypes listType) {
 		this.listType = listType;
 	}
 
+	public boolean getSoftDeleted() { return softDeleted; }
```

Comment:
Method not called.  Consider deleting.

Reply:
I think it would be good for completeness to leave this in since all of the other fields have similar getters and setters defined.

Reply:
Marking resolved - same getter reasoning